docs/usage/custom_cards/custom_card_saxel_fan.md

```diff
@@ -65,7 +65,7 @@ Copy the folder `custom_card_saxel_fan` to `config/minimalist-templates/`
 
 ```yaml
 - type: custom:button-card
-  template: custom_card_saxel_fan_blue
+  template: custom_card_saxel_fan
```

Comment:
This will be a breaking change for those updating their custom cards once in a while